app/templates/components/popover-message.hbs

```diff
@@ -0,0 +1,10 @@
+<i
```

Comment:
I just tried it using voiceover, and navigating the page with keyboard controls only. Below is a screenshot of what voiceover thinks this tooltip is trying to tell me.

![screen shot 2017-03-10 at 4 58 46 pm](https://cloud.githubusercontent.com/assets/2957073/23814789/f210e864-05b2-11e7-9bf4-bd42d6b951da.png)


Accessibility isn't an obvious thing until you've seen it in action once. But it makes a big difference for the users who need it.